### DISCUSSION TITLE
unstable connection of srsue to srs gNB

### DISCUSSION THREAD

**Discussion by SadeghKrmi** [srsran/srsRAN_Project#138] [General Help]
Hello,
I have created a setup using zmq and srsRAN-4G, according to below instruction, my major problem is that srsue is able to connect to gNB just once, and then it is not able to connect, I had to restart both srsgNB and srsUE, then after several times of restarting it is connected. would you please help me here?

https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html

**Comment by andrepuschmann** [comment]
That's expected behavior. We don't support "restarting" just one entity - you have to restart both. The reason is that there is no sync mechanism to realign timing.

**Reply by SadeghKrmi** [reply]
thanks for the response
